lnd_test.go

```diff
@@ -2073,6 +2073,130 @@ func testChannelBalance(net *lntest.NetworkHarness, t *harnessTest) {
 	closeChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
 }
 
+// testChannelUnsettledBalance will test that the UnsettledBalance field
+// is updated according to the number of Pending Htlcs.
+// Alice will send Htlcs to Carol while she is in hodl mode. This will result
+// in a build of pending Htlcs. We expect the channels unsettled balance to
+// equal the sum of all the Pending Htlcs.
+func testChannelUnsettledBalance(net *lntest.NetworkHarness, t *harnessTest) {
+	const chanAmt = btcutil.Amount(1000000)
+	ctxb := context.Background()
+
+	// Create carol in hodl mode.
+	carol, err := net.NewNode("Carol", []string{"--debughtlc", "--hodl.exit-settle"})
```

Comment:
this line looks a little long to me. i'd suggest wrapping the args like:
```golang
carol, err := new.NewNode("Carol", []string{
        "--debughtlc",
        "--hodl.exit-settle",
})
```

Reply:
fixed!